cohorts/varcode_utils.py

```diff
@@ -11,7 +11,12 @@ def __init__(self, variant, variant_collection, patient):
 
     @property
     def variant_metadata(self):
-        return self.variant_collection.metadata[self.variant]
+        source_to_metadata = self.variant_collection.source_to_metadata_dict
```

Comment:
Oh, I didn't understand that we had actually `modified` the `VariantCollection`s `metadata` dictionary in `cohorts`. That's what I was missing 👍